### ISSUE TITLE
BUG: plot(kind='kde') no longer works

### ISSUE THREAD

**Issue by hermanc1** [pydata/pandas#13102]
#### Code Sample, a copy-pastable example if possible
```python
import pandas as pd
import numpy as np
df=pd.DataFrame({'a':np.random.normal(5,1,1000)})
df.a.plot(kind='kde')
```
#### Expected Output
In previous versions, a kde plot was shown. Now I get a "Kernel died unexpectedly" error.
#### output of ``pd.show_versions()``

INSTALLED VERSIONS
------------------
commit: None
python: 3.5.1.final.0
python-bits: 64
OS: Linux
OS-release: 4.4.8-040408-generic
machine: x86_64
processor: x86_64
byteorder: little
LC_ALL: None
LANG: en_ZA.UTF-8

pandas: 0.18.1
nose: 1.3.7
pip: 8.1.1
setuptools: 20.7.0
Cython: 0.24
numpy: 1.11.0
scipy: 0.17.0
statsmodels: None
xarray: None
IPython: 4.2.0
sphinx: 1.4.1
patsy: 0.4.1
dateutil: 2.5.3
pytz: 2016.4
blosc: None
bottleneck: 1.0.0
tables: 3.2.2
numexpr: 2.5.2
matplotlib: 1.5.1
openpyxl: 2.3.2
xlrd: 0.9.4
xlwt: 1.0.0
xlsxwriter: 0.8.5
lxml: 3.6.0
bs4: 4.4.1
html5lib: None
httplib2: None
apiclient: None
sqlalchemy: 1.0.12
pymysql: None
psycopg2: None
jinja2: 2.8
boto: 2.39.0
pandas_datareader: None

**Comment by jreback** [comment]
you either have an installation issue or something else going on.

```
In [1]: pd.__version__
Out[1]: u'0.18.1'

In [2]: np.__version__
Out[2]: '1.11.0'

In [3]: import scipy

In [4]: scipy.__version__
Out[4]: '0.17.0'

In [5]: df=pd.DataFrame({'a':np.random.normal(5,1,1000)})

In [6]: df.a.plot(kind='kde')
Out[6]: <matplotlib.axes._subplots.AxesSubplot at 0x11437bcd0>
```

**Comment by jreback** [comment]
can you plot anything?

**Comment by hermanc1** [comment]
Thank you for your feedback - it put me on the right path. There seems to have been a problem with the MKL library. When I converted my conda install to `nomkl`, using OpenBLAS instead, the problem disappeared. It must have happened when I did a `conda upgrade --all`, updating not only Pandas but also MKL.

BTW, I could plot using kind=`'bar', 'barh', 'line', 'hist', 'box', 'pie'`, but just not `'kde', 'density'`. In the standard Python console (I had been using the IPython console up to this point), I got the following error
Intel MKL FATAL ERROR: Cannot load libmkl_avx2.so or libmkl_def.so.
